feeadjuster/feeadjuster.py

```diff
@@ -183,15 +217,11 @@ def init(options: dict, configuration: dict, plugin: Plugin, **kwargs):
     if plugin.imbalance > 0.5:
         plugin.imbalance = 1 - plugin.imbalance
 
-    plugin.log("Plugin feeadjuster initialized ({} base / {} ppm) with an "
-               "imbalance of {}%/{}%, update_threshold: {}, deactivate_fuzz: {}, adjustment_method: {}"
-               .format(plugin.adj_basefee,
-                       plugin.adj_ppmfee,
-                       int(100 * plugin.imbalance),
-                       int(100 * (1 - plugin.imbalance)),
-                       plugin.update_threshold,
-                       plugin.deactivate_fuzz,
-                       plugin.get_ratio.__name__))
+    plugin.log(f"Plugin feeadjuster initialized ({plugin.adj_basefee} base / {plugin.adj_ppmfee} ppm) with an "
+               f"imbalance of {int(100 * plugin.imbalance)}%/{int(100 * ( 1 - plugin.imbalance))}%, "
+               f"update_threshold: {int(100 * plugin.update_threshold)}%, update_threshold_abs: {plugin.update_threshold_abs}, "
+               f"enough_liquidity: {plugin.big_enough_liquidity}, deactivate_fuzz: {plugin.deactivate_fuzz}, "
+               f"adjustment_method: {plugin.get_ratio.__name__}")
```

Comment:
Much fstring much MSPV 3.6 much wow

Reply:
I've never heard about fstring before you told me about it. I love it! :)